src/routes/safe/store/actions/fetchTransactions.js

```diff
@@ -305,7 +316,7 @@ export type SafeTransactionsType = {
 let etagSafeTransactions = null
 let etagCachedSafeIncommingTransactions = null
 export const loadSafeTransactions = async (safeAddress: string, getState: GetState): Promise<SafeTransactionsType> => {
-  let transactions: TxServiceModel[] = addMockSafeCreationTx(safeAddress)
+  let transactions: TxServiceModel[] = await addMockSafeCreationTx(safeAddress)
```

Comment:
Right now we fetch creation tx everytime we fetch transactions, this will be a problem in the future when we add paginatoin etc.

I think we should fetch it when safe is created/loaded and just add it to the redux store. I am not sure if it will work tho

Not required to do now, but if you have capacity feel free to research/implement it

cc @fernandomg

Reply:
That make sense, but we would need to store that data on localStorage in that case, does that makes sense?

Reply:
I haven't thought about that but honestly I would prefer not to store that, because it can be easily (not as expensive as 300+ tokens) retrieved again and persisting the data complicates things :)

Reply:
So, it's done, after the first fetch is made, the next time it will take it from redux

Reply:
It looks like we didn't understand each other.

What I'm trying to say is that we don't need to fetch creation tx every time we fetch transactions, it doesn't change over time, does it? We should fetch it once the safe is created/loaded and just store it in redux and never touch it anymore. Probably we need a dedicated `fetchSafeCreationTx` action for that

Reply:
Please check this again, we would probably have conflicts with your PR @fernandomg, or maybe not, I added two actions: `fetchSafeCreationTx` and `addOrUpdateTransactions`